api/src/main/resources/hibernate.cfg.xml

```diff
@@ -82,7 +82,6 @@
 		<mapping resource="org/openmrs/api/db/hibernate/EncounterType.hbm.xml" />
         <mapping resource="org/openmrs/api/db/hibernate/EncounterProvider.hbm.xml" />
 		<mapping resource="org/openmrs/api/db/hibernate/EncounterRole.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/Program.hbm.xml" />
                 <mapping resource="org/openmrs/api/db/hibernate/ProgramAttributeType.hbm.xml" />
```

Comment:
we could also have these mapping files on the same line

Reply:
Yes I saw this, but I didn't know if it's ok to modify something that is not in my ticket

Reply:
it is ok to align that